README.md

```diff
@@ -18,6 +18,24 @@ See our full documentation: [http://hermes-pubsub.readthedocs.org/en/latest/](ht
 
 Questions? We are on [gitter](https://gitter.im/allegro/hermes).
 
+## Development
+
+File docker/docker-compose.development.yml disable deployment of hermes frontend, management and consumers.
+
+We have to provide environment (Kafka, ZooKeeper, Graphite, Schema Registry) by command executed in project directory:
```

Comment:
environment -> an environment
by command -> with command
in project -> in the project